Makefile

```diff
@@ -1257,6 +1258,10 @@ conformance-%: $(TEST_ASSET_DIR)/conformance/conformance_test.go
 	go test -mod=mod -ldflags=$(LDFLAGS) -tags conformance -test.v $(TEST_ASSET_DIR)/conformance/... -args $(CONFORMANCE_ARGS) \
 	-run-test=$*
 
+.PHONY: sync-gateway-crds
```

Comment:
```suggestion
.PHONY: sync-gateway-api
```
nit: use same name

Reply:
@jenshu thanks for catching this. Commit https://github.com/solo-io/gloo/commit/3df60f7e1954717aec5ff7466cd34a14e7e4726b resolves this comment.